RELEASE.md

```diff
@@ -33,10 +35,18 @@ $ export GH=/path/to/gh
 
 ## Pre-release tests
 
-Verify the references to the internal Actions by manually running the [release workflow](https://github.com/slsa-framework/slsa-github-generator/actions/workflows/release.yml). Ensure this workflow succeeds. If the references are still `@main`, update them with the following command:
+Verify the references to the internal Actions by manually running the [release workflow](https://github.com/slsa-framework/slsa-github-generator/actions/workflows/release.yml). Ensure this workflow succeeds.
+
+If workflows still refer to `@main`, update them with the following command:
+
+```shell
+find .github/workflows/ -name "*.yaml" -o -name "*.yml" | xargs sed -i "s/uses: slsa-framework\/slsa-github-generator\/\.github\/actions\/\(.*\)@main*/uses: slsa-framework\/slsa-github-generator\/.github\/actions\/\1@$BUILDER_TAG/"
+```
+
+Likewise, if documentation refers to a previous release, update it with the following command:
 
 ```shell
-find .github/workflows/ -name '*.yaml' -o -name '*.yml' | xargs sed -i 's/uses: slsa-framework\/slsa-github-generator\/\.github\/actions\/\(.*\)@main*/uses: slsa-framework\/slsa-github-generator\/.github\/actions\/\1@_YOUR_RELEASE_TAG_/'
+$ find . -name "*.md" -exec sed -i "s/$BUILDER_PREVIOUS_TAG/$BUILDER_TAG/g" {} +
```

Comment:
You may want to rebase with main as I merged #1212 which mostly obviates the need for these changes by adding a more flexible find command.

Reply:
Done.